modules/lambdas/main.tf

```diff
@@ -162,8 +162,8 @@ resource "aws_lambda_function" "request_files" {
       RESTORE_REQUEST_RETRIES  = var.orca_recovery_retry_limit
       RESTORE_RETRY_SLEEP_SECS = var.orca_recovery_retry_interval
       RESTORE_RETRIEVAL_TYPE   = var.orca_recovery_retrieval_type
-	  DB_QUEUE_URL             = var.orca_sqs_status_update_queue_id
-	  ORCA_DEFAULT_BUCKET      = var.orca_default_bucket
+	    DB_QUEUE_URL             = var.orca_sqs_status_update_queue_id
```

Comment:
Can remove the following since they would not be needed anymore.
```
PREFIX                 = var.prefix
DATABASE_PORT   = var.database_port
DATABASE_NAME  = var.database_name
DATABASE_USER   = var.database_app_user
```

Spacing seems odd on new variables added, may want to check.

Reply:
Ah, looks like I had a tab hiding in there.

Reply:
removed